vioscsi/helper.c

```diff
@@ -120,13 +120,16 @@ ENTER_FN_SRB();
             virtqueue_notify(adaptExt->vq[QueueNumber]);
         }
     } else {
+        // virtqueue_add_buf() returned -28 (ENOSPC), i.e. no space for buffer, or some other error
         virtqueue_notify(adaptExt->vq[QueueNumber]);
         ScsiStatus = SCSISTAT_QUEUE_FULL;
         SRB_SET_SRB_STATUS(Srb, SRB_STATUS_BUSY);
         SRB_SET_SCSI_STATUS(Srb, ScsiStatus);
         StorPortBusy(DeviceExtension, 10);
+        RhelDbgPrint(TRACE_LEVEL_WARNING, 
```

Comment:
We're concerned there could be at any point any other sort of error, wouldn't we want to put "res" into this log statement?

Reply:
Yes, absolutely... Will do... 8^d